docs/debugging.md

```diff
@@ -57,6 +57,8 @@ To debug the JavaScript code in Chrome, select "Debug JS Remotely" from the Deve
 
 Select `Tools → Developer Tools` from the Chrome Menu to open the [Developer Tools](https://developer.chrome.com/devtools). You may also access the DevTools using keyboard shortcuts (`⌘⌥I` on macOS, `Ctrl` `Shift` `I` on Windows). You may also want to enable [Pause On Caught Exceptions](http://stackoverflow.com/questions/2233339/javascript-is-there-a-way-to-get-chrome-to-break-on-all-errors/17324511#17324511) for a better debugging experience.
 
+> Note: on Android, if the times between the debugger and device have drifted by more than 60s things might not work or results may not be accurate. Please correct this by running `` adb shell "date `date +%m%d%H%M%Y.%S`" `` on your debugger machine.
```

Comment:
We should also mention @AndrewMorsillo case where animations got messed up even with a slight difference.

Something along the lines of, if you are experiencing unexpected animation or event behavior correct the time.

Reply:
@safaiyeh Working on it. And I might change the wording on `more than 60s` because everything that relies on Timings will be broken if the time is out of sync. what do you think?

Reply:
Sounds good!

Reply:
Another thing to note, I think `adb shelll date` will only work if you have root access to the device in question. Without mentioning that many people could be confused. Also I’m not sure it’s worth mentioning “60s” since the threshold for problems seems to be closer to 1-2 seconds.

Reply:
I also think '60s is the worst-case scenario, and `adb shell date` works well on an emulator but you are correct that when using it on a real device you need to have root access. I will add this to notes section